scripts/remove-redundant-flags.js

```diff
@@ -0,0 +1,187 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import compareVersions from 'compare-versions';
+import esMain from 'es-main';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+
+import bcd from '../index.js';
+import { IS_WINDOWS } from '../test/utils.js';
+
+const dirname = fileURLToPath(new URL('.', import.meta.url));
+
+const getEarliestVersion = (...args) => {
+  const versions = args
+    .filter((version) => typeof version === 'string')
+    .map((version) => version.replace('≤', ''));
+
+  let earliestVersion = versions[0];
```

Comment:
What if `versions` is empty? It will end up returning undefined, but that's an "accident" of how the code is written and doesn't look intentional even if it was. How about:

```suggestion
  let earliestVersion;
```

Then tweak the loop so that earliestVersion is set on the first iteration.